scripts/o3de/o3de/repo_properties.py

```diff
@@ -291,7 +319,9 @@ def edit_repo_props(repo_path: pathlib.Path = None,
     :add_templates: Any template paths to be added to the list
     :delete_templates: Any template names to be removed from the list
     :replace_templates: A list of template paths that will completely replace the current list of path
+    :auto_update: Automatically updates your selected gem/project/template to your remote repository
```

Comment:
This description isn't clear. Looking at it I would assume that this is a boolean argument, which it clearly isn't. 
```suggestion
    :auto_update: List of object types (gem/project/template) to automatically to your remote repository. `None` if no auto update is desired.
```